components/drops.tsx

```diff
@@ -4,9 +4,9 @@ export default function Drops() {
         <div className="drops-section bg-white h-max px-5 py-10">
             <div className="header bg-white">
                 <h1 className="text-black text-4xl font-bold font-sans ml-5">Latest Drops</h1>
-                <div className="bg-purple-500 h-1 mt-2 w-96 ml-5"></div>
+                <div className="bg-purple-500 h-1 mt-2 w-64 md:w-96 ml-5"></div>
             </div>
-            <div className="card-container flex flex-row gap-5 h-96">
+            <div className="card-container flex md:flex-row flex-col gap-5 h-96">
```

Comment:
Making this section flex-col on small screens is causing the overflow issue - you can see how on small screens this is causing a column structure which affects the layout.

Reply:
Then what should I do make, if I make it row then this look little different and rows are not good in small devices